### PR TITLE
Add desired service map

### PR DESCRIPTION
**Pull request by austinlparker**
Contributes to #30 -

- Update service graph diagram with new + modified services
- Add legend

### CODE REVIEW

**Comment by reyang** [comment]
This looks cool @austinlparker 👍🥳

There is one issue with the coloring - legibility.

https://github.com/open-telemetry/opentelemetry-demo-webstore/pull/29#issuecomment-1133530182

image

**Review by reyang — APPROVED** [review]

**Comment by austinlparker** [comment]
Let me play with the text colors a bit to see what I can do.

File: docs/v1Graph.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
+# Desired Service Map
+
+This diagram illustrates the desired relationships between services for the v1 work.
+
+```mermaid
+
```
Comment (reyang) [inline docs/v1Graph.md:6]: nit, remove this empty line

**Comment by reyang** [comment]
> Let me play with the text colors a bit to see what I can do.

Good luck! The tricky thing is to find the colors that work well for both light and dark themes.

**Comment by austinlparker** [comment]
@reyang I tweaked the font colors a bit depending on the language color to improve legibility. I played around with a lot of other options first -- there's some limitations I ran into with the inline style support in mermaid `classDef` (for example, you can't use filter invert greyscale contrast tricks with CSS because you can't do an inline or escaped parentheses in classDefs?). 

The actual colors of the languages are mapped 1:1 from the registry website colors.

**Comment by reyang** [comment]
> @reyang I tweaked the font colors a bit depending on the language color to improve legibility. I played around with a lot of other options first -- there's some limitations I ran into with the inline style support in mermaid `classDef` (for example, you can't use filter invert greyscale contrast tricks with CSS because you can't do an inline or escaped parentheses in classDefs?).
> 
> The actual colors of the languages are mapped 1:1 from the registry website colors.

Looks fantastic!

**Pull request merged**